scripts/submission/submit_to_gencc.py

```diff
@@ -0,0 +1,505 @@
+            submitter_name = "G2P"
+            classification_id = confidence_category[record["confidence"]]
+            classification_name = record["confidence"]
+            dt = datetime.fromisoformat(record["date of last review"])
+            date = dt.strftime("%Y/%m/%d")
+            record_url = "https://www.ebi.ac.uk/gene2phenotype/lgd/" + g2p_id
+            pmids = record["publications"]
+            assertion_criteria_url = "https://www.ebi.ac.uk/gene2phenotype/terminology"
+
+            line_to_output = f"{submission_id}\t{hgnc_id}\t{hgnc_symbol}\t{disease_id}\t{disease_name}\t{moi_id}\t{moi_name}\t{submitter_id}\t{submitter_name}\t{classification_id}\t{classification_name}\t{date}\t{record_url}\t{pmids}\t{assertion_criteria_url}\n"
+            output_file.write(line_to_output)
+            if dry is True:
+                type_of = "create"
+                db_date = create_datetime_now()
+                created_record = create_gencc_submission_record(
+                    submission_id, db_date, type_of, g2p_id
+                )
+                gencc_list.append(created_record)
+            if len(issues_with_record) > 0:
+                with open("record_with_issues.txt", mode="w") as textfile:
+                    for issues in issues_with_record:
+                        textfile.write(f"{issues}\n")
+    
+    return outfile, gencc_list
+
+
+def create_datetime_now() -> date:
+    """Creates datetime at present and formats it to YYYY-MM-DD
+
+    Returns:
+        date: Formatted date
+    """
+
+    time_now = datetime.now()
+    db_date = time_now.strftime("%Y-%m-%d")
+    return db_date
+
+
+def convert_txt_to_excel(input_file: str, output_file: str):
+    """Converts txt to Excel file
+
+    Args:
+        input_file (str): Text file to be converted
+        output_file (str): Excel file that will be created
+    """
+    wb = Workbook()
+    ws = wb.active
+
+    delimiter = "\t"
+
+    with open(input_file, "r") as file:
+        for row_index, line in enumerate(file, start=1):
+            for col_index, value in enumerate(line.strip().split(delimiter), start=1):
+                ws.cell(row=row_index, column=col_index, value=value)
+    wb.save(output_file)
+
+
+def read_from_config_file(config_file: str) -> dict[str, Any]:
+    """Reads from Configuration file (config.ini) and creates a dictionary of data
+
+    Args:
+        config_file (str): configuration file
+
+    Returns:
+        dict[str, Any]: DB/API configuration dict
+    """
+    data = {}
+    config = ConfigParser()
+    config.read(config_file)
+    data["host"] = config["database"]["host"]
+    data["port"] = config["database"]["port"]
+    data["user"] = config["database"]["user"]
+    data["database"] = config["database"]["name"]
+    data["password"] = config["database"]["password"]
+    data["api_url"] = config["api"]["api_url"]
+    data["username"] = config["api"]["username"]
+    data["password"] = config["api"]["password"]
```

Comment:
`data["password"]` is written twice
In the end it saves the API password, not the database.